Clic&Farm-iOSTests/SeedCreationViewTests.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  SeedCreationViewTests.swift
+//  Clic&Farm-iOSTests
+//
+//  Created by Jonathan DE HAAY on 15/11/2018.
+//  Copyright © 2018 Ekylibre. All rights reserved.
+//
+
+import XCTest
+@testable import Clic_Farm_iOS
+
+class SeedCreationViewTests: XCTestCase {
+
+  var addInterventionVC: AddInterventionViewController!
+
+  override func setUp() {
+    super.setUp()
+    let storyboard = UIStoryboard(name: "Main", bundle: Bundle.main)
+    addInterventionVC = storyboard.instantiateViewController(withIdentifier: "AddInterventionVC") as? AddInterventionViewController
+    let _ = addInterventionVC.view
+  }
+
+  override func tearDown() {
+    addInterventionVC = nil
+    super.tearDown()
+  }
+
+  func test_textFieldShouldReturn_shouldReturnFalse() {
+    // Given
+    let textField = UITextField()
+
+    // Then
+    XCTAssertFalse(addInterventionVC.inputsSelectionView.seedView.textFieldShouldReturn(textField),
+                  "Should return false")
```

Comment:
"Should return false" doesn't really inform us about the expected behaviour. Why should it return `false`, what does it mean when it returns true?